control-plane/build-support/functions/10-util.sh

```diff
@@ -620,19 +622,32 @@ function update_version_helm {
 	local prerelease="$3"
 	local full_version="$2"
 	local full_consul_version="$5"
-	if ! test -z "$3"; then
+	local full_consul_dataplane_version="$7"
+	local consul_dataplane_base_path="$8"
+	if ! test -z "$3" && test "$3" != "dev"; then
 		full_version="$2-$3"
+    full_consul_version="$5-$3"
+    full_consul_dataplane_version="$7-$3"
+	elif test "$3" == "dev"; then
+    full_version="$2-$3"
 		# strip off the last minor patch version so that the consul image can be set to something like 1.16-dev. The image
```

Comment:
```suggestion
		full_version="$2-$3"
		full_consul_version="$5-$3"
		full_consul_dataplane_version="$7-$3"
	elif test "$3" == "dev"; then
		full_version="$2-$3"
		# strip off the last minor patch version so that the consul image can be set to something like 1.16-dev. The image
```